app/controllers/home_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
       endpoint: 'https://sandbox.iexapis.com/v1'
     )
 
-    @symbols = @client.ref_data_symbols
+    # @symbols = @client.ref_data_symbols
```

Comment:
Ma'am, if you see this, this is not an unnecessary comment. We only commented this to temporarily speed up page loading times 😅